src/simulation/dynamics/facetSRPDynamicEffector/facetSRPDynamicEffector.rst

```diff
@@ -21,168 +23,186 @@ provides information on what this message is used for.
+
+    # Define facet articulation axes in B frame components
+    rotAxes_B = [np.array([0.0, 0.0, 0.0]),
+                 np.array([0.0, 0.0, 0.0]),
+                 np.array([0.0, 0.0, 0.0]),
+                 np.array([0.0, 0.0, 0.0]),
+                 np.array([0.0, 0.0, 0.0]),
+                 np.array([0.0, 0.0, 0.0]),
+                 np.array([1.0, 0.0, 0.0]),
+                 np.array([1.0, 0.0, 0.0]),
+                 np.array([-1.0, 0.0, 0.0]),
+                 np.array([-1.0, 0.0, 0.0])]
+
+    # Define facet optical coefficients
     specCoeff = np.array([0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.9, 0.9])
     diffCoeff = np.array([0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1])
 
-#. Populate the scGeometry structure with the defined facet information::
+.. important::
+    Note that in order to use this module, the facet articulation axes must always be configured regardless of whether
+    articulated facets are considered. For all static facets, the articulation axes must be set to zero. Ensure that the
+    specified number of articulated facets matches the number of nonzero articulation axes.
+
+.. important::
+    The module requires the articulated facet data to be added at the end of the facet data vectors.
 
-    for i in range(len(facetAreas)):
-        newSRP.addFacet(facetAreas[i], specCoeff[i], diffCoeff[i], normals_B[i], locationsPntB_B[i])
+#. Populate the module's ``FacetedSRPSpacecraftGeometryData`` structure with the spacecraft facet information using the ``addFacet()`` method::
 
-Note that for each added facet, the user is required to set the corresponding area, normal vector, location, and optical coefficients parameters in the scGeometry structure.
+    for i in range(numFacets)):
+        SRPEffector.addFacet(facetAreas[i], specCoeff[i], diffCoeff[i], facetNormals_B[i], locationsPntB_B[i], rotAxes_B[i])
 
 #. Connect the Sun's ephemeris message to the SRP module::
 
-    newSRP.sunInMsg.subscribeTo(sunMsg)
+    SRPEffector.sunInMsg.subscribeTo(sunMsg)
 
 #. Add the SRP dynamic effector to the spacecraft::
 
-    scObject.addDynamicEffector(newSRP)
+    scObject.addDynamicEffector(SRPEffector)
 
    See :ref:`spacecraft` documentation on how to set up a spacecraft object.
 
-#. Set up the spacecraft orbit::
-
-    oe = orbitalMotion.ClassicElements()
-    r_eq = 6371*1000.0  # [m]
-    rN = np.array([r_eq+2000.0, -149598023 * 1000, 0.0])  # [m]
-    vN = np.array([0.0, 7.90854, 0.0])  # [m/s]
-    sig_BN = np.array([0.0, 0.0, 0.0])
-
-#. Initialize the spacecraft states with the initialization variables::
-
-    scObject.hub.r_CN_NInit = rN  # [m] r_CN_N
-    scObject.hub.v_CN_NInit = vN  # [m] v_CN_N
-    scObject.hub.sigma_BNInit = sig_BN
-
-#. Add the module to the task list::
+#. Finally, add the SRP effector module to the task list::
 
-    unitTestSim.AddModelToTask(unitTaskName, newSRP)
+    unitTestSim.AddModelToTask(unitTaskName, SRPEffector)

```

Comment:
At the end of the module documentation, please add a statement that the updated SEP Momentum Management scenario is an illustration of using articulated panels with SRP calculation.

Reply:
Done